src/dispatchers/keyEventDispatcher.ts

```diff
@@ -0,0 +1,67 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Dispatcher {
+  export class KeyEvent {
```

Comment:
Not sure about KeyEvent (since Mouse then should be called MouseEvent.... which also leads me to believe that the Event part is redundant)